glean/load.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// clang-format off
+// This file exposes a list of metrics/pings to the QML/JS world.
+.pragma library
+// Importing Glean.js as released by the Glean team.
+.import 'glean.js' as RealGlean
+// clang-format on
+
+const _glean = RealGlean.Glean.default;
+
+// Initialization.
+_glean.initialize('MozillaVPN', true, {debug: {logPings: true}});
+
+// TODO: I would like to use glean_parser, but the output is meant to be used
+// by webextensions only (the `import DatetimeMetricType from
+// "@mozilla/glean/webext/private/metrics/datetime"` would not work here). Is
+// OK to create the list of metrics/pings manually in this way?
```

Comment:
Yes, for now this is way. I suppose you saw this already, but this is what we did in our own Qt [sample](https://github.com/mozilla/glean.js/blob/main/samples/qt-qml-app/pings.js) as well.